### DISCUSSION TITLE
댓글 개수 보이기

### DISCUSSION THREAD

**Discussion by Taaewoo** [pronist/hELLO#124] [Ideas]
안녕하세요 블로그 스킨 정말 유용하게 잘 사용하고 있습니다!

스킨을 이용하던 중 아래의 내용이 반영된다면 좋을 것 같아서 이렇게 discussion을 작성하게 되었습니다.

#112 
기존 discussion 중 대댓글 펼치기 기능 추가에 대한 내용이 있었는데 비슷한 내용이지만 살짝 다른 구현입니다!

## Discussion
- 게시물의 대댓글 갯수가 보였으면 좋겠습니다

현재는 대댓글이 달리더라도 "댓글보기"를 눌러야만 대댓글을 확인할 수가 있습니다.

이렇다보니 지금 현재 대댓글이 달린건지 아닌지 직접 눌러보기 전까지는 알 수가 없더라구요.

<img width="210" alt="image" src="https://user-images.githubusercontent.com/28804154/184385075-7eea29b5-e448-46e3-a405-44982fc558de.png">

그래서 대댓글 펼치기 옵션을 검토중이라고 말씀해주셨는데, 펼치기 옵션 패치가 어렵다면 현재 대댓글 갯수라도 보였으면 좋겠습니다.

"댓글보기(2)" 이렇게요!

**Comment by pronist** [comment]
@Taaewoo 안녕하세요.

아이디어 감사드립니다. 다음 패치 때 고려해보도록 하겠습니다. UX 관점에서 좋을 것 같습니다. :)
